test/load/src/main.rs

```diff
@@ -49,6 +127,49 @@ async fn get_recommendations(user: &mut GooseUser) -> TransactionResult {
     Ok(())
 }
 
+// async fn stop_recommending(user: &mut GooseUser) -> TransactionResult {
+//     let listing_id = rand::thread_rng().gen_range(1..=2000);
+//     let goose = user
+//         .get(&format!("/api/recommendations/stop/{}", listing_id))
+//         .await?;
+
+//     let validate = &Validate::builder().status(200).build();
+//     validate_page(user, goose, validate).await?;
+
+//     Ok(())
+// }
+
+async fn search_listings(user: &mut GooseUser) -> TransactionResult {
+    let query = Sentence(3..5).fake::<String>();
+    let latitude = rand::thread_rng().gen_range(-90.0..=90.0);
+    let longitude = rand::thread_rng().gen_range(-180.0..=180.0);
+
+    let url = format!(
+        "/api/search?query={}&latitude={}&longitude={}",
+        query, latitude, longitude
+    );
+    let goose = user.get(&url).await?;
+
+    let validate = &Validate::builder()
+        .status(200)
+        .texts(vec!["items", "totalItems"])
+        .build();
+    validate_page(user, goose, validate).await?;
+
+    Ok(())
+}
+
+async fn get_search_history(user: &mut GooseUser) -> TransactionResult {
+    let goose = user.get("/api/1/search-history").await?;
+
+    let validate = &Validate::builder().status(200).build();
+    validate_page(user, goose, validate).await?;
+
+    Ok(())
+}
+
+// FRONTEND
+
```

Comment:
For the below `get_index` test, lets use [`validate_and_load_static_assets`](https://docs.rs/goose-eggs/latest/goose_eggs/fn.validate_and_load_static_assets.html#)